frontend/src/component/common/SegmentItem/SegmentItem.tsx

```diff
@@ -25,7 +25,9 @@ const StyledAccordion = styled(Accordion, {
     shouldForwardProp: (prop) => prop !== 'isDisabled',
 })<{ isDisabled: boolean | null }>(({ theme, isDisabled }) => ({
     border: `1px solid ${theme.palette.divider}`,
-    borderRadius: theme.shape.borderRadiusMedium,
+    '&.segment-accordion': {
+        borderRadius: theme.shape.borderRadiusMedium,
+    },
```

Comment:
Increase the specificity of this selector to override the default for mui accordions (mentioned in the PR description).

We could also use the [`:is` specificity trick](https://css-tricks.com/where-has-a-cool-specificity-trick-too/) and do something like this to avoid adding the class:
```ts
    '&:is(*,#selector-for-specificity)': {
        borderRadius: theme.shape.borderRadiusMedium,
    },
```

But I think using a class is clearer.

Reply:
Yes i agree, i think its clearer with a class